src/cdi/CPCommon.hh

```diff
@@ -20,9 +20,10 @@ namespace constants {
 unsigned int const num_CPModels(3);
 } // namespace constants
 
-enum class CPModel {
-  ELOSS = 0,        /*!< energy loss model */
-  LA_SCATTERING = 1 /*!< large-angle scattering model */
+enum class CPModelAngleCutoff {
+  NONE = 0, /*!< no angle cutoff */
+  TNBURN =
+      1 /*!< cutoff angle formula from "TN Burn Project: Minimum Impact parameter Scattering Angle Cutoff/Nuclear Interactions */
```

Comment:
Please wrap this comment to keep it below 80 cols.  See example at https://github.com/lanl/Draco/pull/723/files